buildSrc/src/main/kotlin/com.hedera.hashgraph.jpms-modules.gradle.kts

```diff
@@ -23,21 +23,20 @@ plugins {
 
 extraJavaModuleInfo {
     failOnMissingModuleInfo.set(true)
-    
+
     automaticModule("com.hedera.hashgraph:protobuf-java", "com.hedera.hashgraph.protobuf.java.api")
 
     automaticModule("com.goterl:lazysodium-java", "lazysodium.java")
     automaticModule("com.goterl:resource-loader", "resource.loader")
 
     automaticModule("io.grpc:grpc-api", "io.grpc.api")
     automaticModule("io.grpc:grpc-core", "io.grpc.core")
-    automaticModule("io.grpc:grpc-netty", "io.grpc.netty")
+    automaticModule("io.grpc:grpc-netty", "grpc.netty")
```

Comment:
If you do a merge of develop in your branch this should already be fixed based on https://github.com/hashgraph/hedera-services/pull/4540/files

Reply:
merged in develop